src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -58,52 +56,48 @@
 import org.reactome.addlinks.referencecreators.NCBIGeneBasedReferenceCreator;
 import org.reactome.addlinks.referencecreators.OneToOneReferenceCreator;
 import org.reactome.addlinks.referencecreators.RHEAReferenceCreator;
+import org.reactome.addlinks.referencecreators.TargetPathogenReferenceCreator;
 import org.reactome.addlinks.referencecreators.UPMappedIdentifiersReferenceCreator;
-import org.reactome.release.common.database.InstanceEditUtils;
 
 
 public class AddLinks
 {
+	private static final String PROTEIN = "PROTEIN";
+	private static final String CTD_PROCESSOR = "CTDProcessor";
+	private static final String BRENDA_UC = "BRENDA";
+	private static final String KEGG = "KEGG";
+	private static final String BRENDA = "Brenda";
+	private static final String ENSEMBL = "ENSEMBL";
+	private static final String REACTOME = "Reactome";
+	private static final String HUMAN_SPECIES_ID = "48887";
```

Comment:
I don't quite know how AddLinks accesses these values, whether they're only used in this file or elsewhere, but depending on if this list grows maybe we could consider an AddLinksConstants class at some point in the future?

Reply:
If it makes sense. Most constants are declared in the context in which they are used. If it looks like there are many constants being used across multiple classes, this is something that could be considered.